modules/mbedtls/Kconfig

```
@@ -111,6 +112,37 @@ config MBEDTLS_DEBUG_LEVEL
 	  This makes Zephyr call mbedtls_debug_set_threshold() function during
 	  mbedTLS initialization, with the configured debug log level.
 
+choice MBEDTLS_DEBUG_EXTRACT_BASENAME
+	prompt "Extract basename from filenames"
+	default MBEDTLS_DEBUG_EXTRACT_BASENAME_AT_BUILDTIME
+
+config MBEDTLS_DEBUG_EXTRACT_BASENAME_AT_BUILDTIME
+	bool "At buildtime"
```

Just `buildtime`, looks cleaner in menuconfig as selected choice in the
 `Extract basename from filenames (buildtime)`
```suggestion
	bool "buildtime"
```